scripts/build-downstream-projects.sh

```diff
@@ -105,5 +105,5 @@ EOF
 
 
 _ example_helloworld
-# _ spl
+_ spl
```

Comment:
:eyes: no wonder pure me didn't notice the expected spl cli breakages even if I waited for my last. here was dragons. :)